Makefile

```
@@ -22,6 +22,7 @@ benchmark:
 
 analyze:
 	go vet ./...
+	gosec -quiet -tests ./...
```

The `-tests` flag actually **enables** [gosec](https://github.com/securego/gosec) to run on test files, which is what we don't want.

```suggestion
	gosec -quiet ./...
```